src/RA_Dlg_Memory.cpp

```diff
@@ -1777,10 +1781,17 @@ bool Dlg_Memory::GetGameMemoryRange(ra::ByteAddress& start, ra::ByteAddress& end
 
 static TCHAR* ParseAddress(TCHAR* ptr, ra::ByteAddress& address) noexcept
 {
+    Expects(ptr != nullptr);
```

Comment:
Can't use `gsl::not_null` with pointer arithmetic so we gotta check for nullness on all paths.